evaluation/gorilla/run_infer.py

```diff
@@ -167,6 +167,9 @@ async def process_instance(
     dataset = prepare_dataset(
         dataset_df, output_file=output_file, eval_n_limit=args.eval_n_limit
     )
+    url = 'https://raw.githubusercontent.com/ShishirPatil/gorilla/main/eval/eval-scripts/codebleu/parser/my-languages.so'
```

Comment:
Can you add a if-statement to try to detect whether the file first exists? If so, we should skip?